security-command-center/snippets/pom.xml

```diff
@@ -80,4 +92,13 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+	  <build>
+	  <plugins>
+	  <plugin>
+	  <groupId>org.apache.maven.plugins</groupId>
+	  <artifactId>maven-surefire-plugin</artifactId>
+ 	  <version>3.2.5</version>
+	  </plugin>
+	  </plugins>
+	  </build>
```

Comment:
nit: please format the XML of the build plugin

Reply:
Addressed